doc/input-output-reference/src/overview/group-internal-gains-people-lights-other.tex

```diff
@@ -301,6 +285,22 @@ \subsubsection{Inputs}\label{inputs-025}
 
 The following IDF example allows for a maximum of 31 people with scheduled occupancy of ``Office Occupancy'', 60\% radiant using an Activity Schedule of ``Activity Sch''. The example allows for thermal comfort reporting.
 
+\paragraph{Field: Cold Stress Temperature Threshold}\label{cold-stress-temperature-threshold}
+
+This field is used in the calculation of hours of safety for cold events, which
+is defined as the longest duration (number of hours), starting from the
+beginning time of the risk period (e.g., the start time of a power outage) to
+not below a certain temperature threshold (specified in this field). If not
+specified, 60$^\circ$F (or 15.56$^\circ$C) will be used as the default temperature threshold.
+
+\paragraph{Field: Heat Stress Temperature Threshold}\label{heat-stress-temperature-threshold}
+
+This field is used in the calculation of hours of safety for hot events, which
+is defined as the longest duration (number of hours), starting from the
+beginning time of the risk period (e.g., the start time of a power outage) to
+not above a certain temperature threshold (specified in this field). If not
+specified, 86$^\circ$F (or 30$^\circ$C) will be used as the default temperature threshold.
```

Comment:
So these were moved down here?  It kinda feels a bit odd given the paragraph text above saying the IDF text follows.